RELEASE/scripts/sl_ascend/sl_mr2018.ash

```diff
@@ -440,7 +440,7 @@ boolean songboomSetting(int option)
 
 int catBurglarHeistsLeft()
 {
-	if (!have_familiar($familiar[Cat Burglar]) !sl_is_valid($familiar[Cat Burglar]) || !is_unrestricted($item[kitten burglar]))
+	if (!have_familiar($familiar[Cat Burglar]) || !sl_is_valid($familiar[Cat Burglar]) || !is_unrestricted($item[kitten burglar]))
```

Comment:
`sl_is_valid` already checks `is_unrestricted`, so no need to do that here as well.